### PR TITLE
{2023.06}[GCCcore/12.3.0-GCC/12.3.0-gompi/2023a] bio-packages

### PR DESCRIPTION
**Pull request by TopRichard**
```
missing packages :

BWA-0.7.17-GCCcore-12.3.0.
CapnProto-1.0.1-GCCcore-12.3.0
DendroPy-4.6.1-GCCcore-12.3.0
DIAMOND-2.1.8-GCC-12.3.0 
FastME-2.1.6.3-GCC-12.3.0
fastp-0.23.4-GCC-12.3.0
HMMER-3.4-gompi-2023a
IQ-TREE-2.3.5-gompi-2023a
KronaTools-2.8.1-GCCcore-12.3.0
LSD2-2.4.1-GCCcore-12.3.0
MAFFT-7.520-GCC-12.3.0-with-extensions
ncbi-vdb-3.0.10-gompi-2023a
```

### CODE REVIEW

**Comment by TopRichard** [comment]
bot: build repo:eessi.io-2023.06-software arch:x86_64/generic
bot: build repo:eessi.io-2023.06-software arch:x86_64/intel/haswell
bot: build repo:eessi.io-2023.06-software arch:x86_64/intel/skylake_avx512
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen2
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen3
bot: build repo:eessi.io-2023.06-software arch:aarch64/generic
bot: build repo:eessi.io-2023.06-software arch:aarch64/neoverse_n1
bot: build repo:eessi.io-2023.06-software arch:aarch64/neoverse_v1

File: easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.2-2023a.yml [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
+  - fastp-0.23.4-GCC-12.3.0.eb
+  - HMMER-3.4-gompi-2023a.eb
+  - IQ-TREE-2.3.5-gompi-2023a.eb:
+      options:
+        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/20955
+        from-commit: 185f88b9a03d65a7fb74edc7acb4221e87e90784      
+  - KronaTools-2.8.1-GCCcore-12.3.0.eb
+  - LSD2-2.4.1-GCCcore-12.3.0.eb
+  - MAFFT-7.520-GCC-12.3.0-with-extensions.eb
+  - ncbi-vdb-3.0.10-gompi-2023a.e
```
Comment (bedroge) [inline easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.2-2023a.yml:17]:
```suggestion
  - ncbi-vdb-3.0.10-gompi-2023a.eb
```

**Comment by TopRichard** [comment]
bot: build repo:eessi.io-2023.06-software arch:x86_64/generic
bot: build repo:eessi.io-2023.06-software arch:x86_64/intel/haswell
bot: build repo:eessi.io-2023.06-software arch:x86_64/intel/skylake_avx512
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen2
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen3
bot: build repo:eessi.io-2023.06-software arch:aarch64/generic
bot: build repo:eessi.io-2023.06-software arch:aarch64/neoverse_n1
bot: build repo:eessi.io-2023.06-software arch:aarch64/neoverse_v1

**Comment by bedroge** [comment]
All builds actually completed, but it failed to create the tarball. Looks like it's due to some changes from #618.

**Comment by TopRichard** [comment]
bot: build repo:eessi.io-2023.06-software arch:x86_64/generic
bot: build repo:eessi.io-2023.06-software arch:x86_64/intel/haswell
bot: build repo:eessi.io-2023.06-software arch:x86_64/intel/skylake_avx512
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen2
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen3
bot: build repo:eessi.io-2023.06-software arch:aarch64/generic
bot: build repo:eessi.io-2023.06-software arch:aarch64/neoverse_n1
bot: build repo:eessi.io-2023.06-software arch:aarch64/neoverse_v1

**Comment by bedroge** [comment]
bot: build repo:eessi.io-2023.06-software arch:x86_64/generic

**Comment by bedroge** [comment]
bot: build repo:eessi.io-2023.06-software arch:aarch64/generic
bot: build repo:eessi.io-2023.06-software arch:aarch64/neoverse_n1
bot: build repo:eessi.io-2023.06-software arch:aarch64/neoverse_v1
bot: build repo:eessi.io-2023.06-software arch:x86_64/generic
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen2
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen3
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen4
bot: build repo:eessi.io-2023.06-software arch:x86_64/intel/haswell
bot: build repo:eessi.io-2023.06-software arch:x86_64/intel/skylake_avx512

**Comment by bedroge** [comment]
The zen2 build failed due to a failing HMMER test:
```
    exercise   18 [ p7_hmmd_search_stats] ...     Aborted (core dumped)
FAILED [crash!]
```

**Comment by bedroge** [comment]
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen2

**Review by bedroge — APPROVED** [review]

**Pull request merged**